docs/a320-simvars.md

```diff
@@ -360,30 +360,30 @@
 - A32NX_APU_EGT_CAUTION
     - Celsius
     - The APU's exhaust gas temperature caution level, to be indicated in amber in the cockpit,
-      when f64::MIN/-Number.MAX_VALUE the ECB isn't supplying information, for example due to being unpowered.
+      when < -273.15 the ECB isn't supplying information, for example due to being unpowered.
```

Comment:
Well, we are in trouble if the ECB gets that cold. At least we won't have to worry about it.

Reply:
I think the whole universe would have collapsed by that point. 😀